fastify.js

```diff
@@ -26,6 +26,8 @@ const runHooks = require('./lib/hookRunner')
 
 const DEFAULT_JSON_BODY_LIMIT = 1024 * 1024 // 1 MiB
 
+const childrenKey = Symbol('children')
```

Comment:
How about `Symbol.for('FASTIFY_CHILDREN')`?

Reply:
it's a private Symbol, not sure if it's needed, but ok as well.

Reply:
/shrug

Reply:
If we use `Symbol.for`, it doesn't seem like there would be much point to using a Symbol since users could still access the property.